tests/csm/rest/test_iam_users.py

```diff
@@ -1430,3 +1430,187 @@ def test_37783(self):
         assert_utils.assert_true(len(get_resp.json()["keys"]) == 1,
                                  "Number of Access keys are not matching")
         self.log.info("##### Test completed -  %s #####", test_case_name)
+
+    @pytest.mark.csmrest
+    @pytest.mark.lc
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-38087')
+    def test_38087(self):
+        """
+        Update User with display name, correct uid in request.
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("[START] STEP 1: Creating IAM user with basic parameters")
+        payload = self.csm_obj.iam_user_payload_rgw("random")
+        self.log.info("payload :  %s", payload)
+        resp = self.csm_obj.create_iam_user_rgw(payload)
+        assert_utils.assert_true(resp.status_code == HTTPStatus.CREATED, "IAM user creation failed")
+        uid = resp.json()["tenant"] + "$" + payload['uid']
+        self.created_iam_users.add(uid)
+        self.log.info("STEP 2: Perform get iam users")
+        get_resp = self.csm_obj.get_iam_user(uid)
+        assert_utils.assert_true(get_resp.status_code == HTTPStatus.OK, "Get IAM user failed")
+        self.log.info("STEP 3: Send patch request to update display name")
+        payload = {}
+        payload.update({"display_name":uid})
```

Comment:
use different modified uid for display name modification

Reply:
done